Content.Shared/Body/Components/BodyComponent.cs

```diff
@@ -2,28 +2,53 @@
 using Content.Shared.Body.Prototypes;
 using Content.Shared.Body.Systems;
 using Robust.Shared.Audio;
+using Robust.Shared.Containers;
 using Robust.Shared.GameStates;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using System;
+using Robust.Shared.Serialization;
 
 namespace Content.Shared.Body.Components;
 
 [RegisterComponent, NetworkedComponent]
 [Access(typeof(SharedBodySystem))]
 public sealed partial class BodyComponent : Component
 {
-    [DataField("prototype", customTypeSerializer: typeof(PrototypeIdSerializer<BodyPrototype>))]
+    [DataField("prototype", customTypeSerializer: typeof(PrototypeIdSerializer<BodyPrototype>)), AutoNetworkedField]
```

Comment:
this is marked autonetworked but without autogeneratecompstate, i assume this isnt meant to be auto generated since body is manually handled

Reply:
good catch